packages/starlight/translations/ar.json

```
@@ -0,0 +1,22 @@
+{
+  "skipLink.label": "الذهاب للمحتوى",
+  "search.label": "البحث",
+  "search.shortcutLabel": "(إضغط / للبحث)",
+  "search.cancelLabel": "إلغاء",
+  "search.devWarning": "البحث متوفر فقط في نسخ الاطلاق. \n جرب بناء المشروع ومعاينته على جهازك",
+  "themeSelect.accessibleLabel": "إخت ثيم",
```

Is there a typo here?
```suggestion
  "themeSelect.accessibleLabel": "إختر ثيم",
```